spec/spec_helper.rb

```diff
@@ -27,10 +12,15 @@
     .to_return(status: 200, body: WebmockStubs.mock_artwork, headers: {})
     stub_request(:get, 'https://www.localhost:3000/api/v1/users/1/recommendations')
     .to_return(status: 200, body: WebmockStubs.mock_artwork_recommendations, headers: {})
-    # stub_request(:get, 'https://www.localhost:3000/api/v1/art/4')
-    # .to_return(status: 200, body: WebmockStubs.mock_artwork_recommendations, headers: {})
+    stub_request(:get, 'https://www.localhost:3000/api/v1/art/4')
+    .to_return(status: 200, body: WebmockStubs.mock_artwork_recommendations, headers: {})
+    stub_request(:get, 'https://www.localhost:3000/api/v1/users/1/rated_art/4')
+    .to_return(status: 200, body: WebmockStubs.mock_artwork_recommendations, headers: {})
     stub_request(:get, 'https://www.localhost:3000/api/v1/art/1')
     .to_return(status: 200, body: WebmockStubs.mock_artwork_recommendations, headers: {})
+    stub_request(:get, 'https://www.localhost:3000/api/v1/users/1/rated_art/1')
```

Comment:
ooh there might be a conflict here we changed all the urls to connect to our backend peaceful-reef heroku. Definitely something to keep an eye out for in merge.